ROADMAP.md

```diff
@@ -0,0 +1,54 @@
+- ⬆️🏃 Teach conda create  how to handle environment.yml files (#14113)
+- ⬆️🏃 Add support for conda [env] hooks (#13833).
+- ⬆️📪 Implement the aforementioned hooks in conda-pypi
+- ⬆️📪 Define and standardize a conda lockfile schema.
+- ⬆️📪 Replace conda-lock with an officially supported conda plugin for lockfile generation.
+- 🏃 ~40% of environment build time is due to package download and extraction, which is carried out serially to avoid collisions when multiple concurrent requests for the same package are made. Implement a locking mechanism to allow concurrent downloads, reducing this to just the bare network time required to download the required assets.
+- 🏃 Currently, simultaneous calls to conda create result in corrupted packages and cached repo data (#13037). Support for locking the repodata cache was added by the community recently (#12996), but full locking support must be added to prevent conflicts/data corruption from other concurrent conda operations.
+- ⬆️📪 Add native conda support for locking to allow parallel conda commands on a single machine (#13055).
+
+## Storage / Backup and Restore
+
+- 📪 A major concern for existing conda-store deployments is that old, unused builds still take up space on shared volumes. Old builds eventually fill up disk space for deployments with many users, requiring manual intervention to delete them. Implementing a backup mechanism will require reaching consensus on an approach to archiving and recreation old builds (#889).
+- 📪 Implement archiving of old builds: the yaml spec and the lockfile are kept, but the environments on disk are deleted.
+- 📪 Implement bulk archiving of old builds. --all-but-last, --older-than=<date>.
+- 📪 Implement recreation of old builds. Combined with planned performance improvements, users can expect to wait only as long as it takes to download and reinstall the packages for the recreated environment.
+- 📪 Transferring or copying a full conda-store deployment is currently not possible. Implement full backup and restore, including creating the active environments on disk.
+- 📪 Currently, conda environments created by conda-store do not share packages. In deployments containing many builds, this can lead to many duplicate packages, which uses much more storage than necessary. Refactor the logic around creating new environments so that packages are shared across environments.
+- 📪 Design a serialization format for conda-store environments that captures both the history of specfiles and lockfiles to allow transferring and copying of individual environments.
+
+## User experience improvements
+
+- 🏃 Currently there are three ways to interact with the conda-store-server: the CLI, the REST API, and the Python API. However, these interfaces do not provide the same functionality, leading to maintenance issues. Addressing this will require refactoring the existing CLI.
```

Comment:
Same, the bullets below need to be appropriately formatted/indented to fall under this main bullet point